pkgs/development/python-modules/scenedetect/default.nix

```diff
@@ -0,0 +1,72 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  av,
+  click,
+  numpy,
+  pytestCheckHook,
+  opencv-python,
+  platformdirs,
+  tqdm,
+}:
+let
+  testsResources = fetchFromGitHub {
+    owner = "Breakthrough";
+    repo = "PySceneDetect";
+    rev = "resources";
+    hash = "sha256-7ws7F7CkEJAa0PgfMEOwnpF4Xl2BQCn9+qFQb5MMlZ0=";
+  };
+in
+buildPythonPackage rec {
+  pname = "scenedetect";
+  version = "0.6.6";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Breakthrough";
+    repo = "PySceneDetect";
+    tag = "v${version}-release";
+    hash = "sha256-G5NLk6eOpclfrzzHad2KT3uZqydSJU0oF/4L2NIdZe0=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    av
+    click
+    numpy
+    opencv-python
+    platformdirs
+    tqdm
+  ];
+
+  pythonImportsCheck = [
+    "scenedetect"
+  ];
+
+  preCheck = ''
+    ln -s ${testsResources}/tests/resources tests/
+  '';
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
```

Comment:
```suggestion
    versionCheckHook
  ];
  versionCheckProgramArg = "version";
```

Reply:
Good to know! I've never knew this existed. Found it deep in the nixpkgs manual.